src/reducers/versions.spec.tsx

```diff
@@ -779,6 +779,29 @@ describe(__filename, () => {
       });
     });
 
+    it('calls getVersion with a given path', async () => {
+      const version = fakeVersion;
+      const path = 'some/file.js';
+      const _getVersion = jest.fn().mockReturnValue(Promise.resolve(version));
+
+      const addonId = 123;
+      const versionId = version.id;
+
+      const { store, thunk } = thunkTester({
+        createThunk: () =>
+          fetchVersion({ _getVersion, addonId, versionId, path }),
+      });
+
+      await thunk();
+
+      expect(_getVersion).toHaveBeenCalledWith({
+        addonId,
+        apiState: store.getState().api,
+        path,
```

Comment:
For tests like these I think it's nice to limit the assertion to the specific thing you're testing. In this case it would be:

```tsx

      expect(_getVersion).toHaveBeenCalledWith(
        expect.objectContaining({
          path,
        }),
      );
```

With that, you can remove unnecessary setup and it will also allow the call signature of `fetchVersion()` to evolve in unrelated ways without breaking this test.